app/uk/gov/hmrc/crsfatcaregistration/connectors/SubscriptionConnector.scala

```diff
@@ -48,12 +48,10 @@ class SubscriptionConnector @Inject() (
     subscription: DisplaySubscriptionRequest
   )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] = {
     val serviceName = "read-subscription"
-    http.POST[DisplaySubscriptionRequest, HttpResponse](
+    http.GET[HttpResponse](
       config.baseUrl(serviceName),
```

Comment:
The URL should include the id type and the id value - see 11.2 and 11.3 of the spec.